### PR TITLE
Revert "ISSUE-55 - Memory leaks from requests causing completionBlock to be retained indefinitely"

### PR DESCRIPTION
**Pull request by RicoYao**
This reverts commit 89849ae50129acb8f8a7c16caefa6976e0e6cac2.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @RicoYao has signed the CLA. Thanks for the pull request!

File: BoxContentSDK/BoxContentSDK/Operations/BOXAPIAppUsersAuthOperation.m [hunk 1]
```diff
@@ -48,9 +48,9 @@ - (void)executeOperation
                     weakSelf.accessToken = accessToken;
                     weakSelf.accessTokenExpiration = accessTokenExpiration;
                 } else if (!error) {
-                    weakSelf.error = [[NSError alloc] initWithDomain:@"accessToken and accessTokenExpiration should be non-nil" code:BOXContentSDKAppUserErrorAccessTokenInvalid | BOXContentSDKAppUserErrorAccessTokenExpirationInvalid userInfo:nil];
+                    self.error = [[NSError alloc] initWithDomain:@"accessToken and accessTokenExpiration should be non-nil" code:BOXContentSDKAppUserErrorAccessTokenInvalid | BOXContentSDKAppUserErrorAccessTokenExpirationInvalid userInfo:nil];
```
Comment (boxtcarpel) [inline BoxContentSDK/BoxContentSDK/Operations/BOXAPIAppUsersAuthOperation.m:51]: Should these actually be weakSelf? Otherwise we're mixing in this block.

File: BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleFileDetailsController.m [hunk 2]
```diff
@@ -45,7 +45,6 @@ - (void)viewDidLoad
     if ([self.itemType isEqualToString:BOXAPIItemTypeFile]) {
         
         BOXFileRequest *request = [self.client fileInfoRequestWithID:self.itemID];
-
```
Comment (boxtcarpel) [inline BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleFileDetailsController.m:48]: undo this change so the file's commit history isn't sullied?

**Comment by boxtcarpel** [comment]
+2

1 real comment.

:japanese_ogre:
